From mithlesh@linsyssoft.com  Fri Jun  5 11:17:53 2009
From: Mithlesh Thukral <mithlesh@linsyssoft.com>
Date: Fri, 5 Jun 2009 15:33:59 +0530 (IST)
Subject: Staging: wlan-ng: Update README file. Remove Lindent task
To: Greg Kroah-Hartman <gregkh@suse.de>
Message-ID: <alpine.LFD.2.00.0906051533170.16520@localhost.localdomain>


Remove Lindent changes as a TODO item in the wlan-ng directory of staging tree.

Signed-off-by: Mithlesh Thukral <mithlesh@linsyssoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/wlan-ng/README |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/staging/wlan-ng/README
+++ b/drivers/staging/wlan-ng/README
@@ -1,7 +1,6 @@
 TODO:
 	- checkpatch.pl cleanups
 	- sparse warnings
-	- Lindent cleanups
 	- move to use the in-kernel wireless stack
 
 Please send any patches or complaints about this driver to Greg
